xml/System.Diagnostics/Activity.xml

```diff
@@ -379,9 +387,17 @@ In .NET 5.0 and later, the default format is <xref:System.Diagnostics.ActivityId
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the display name of the activity.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+`DisplayName` is intended to be used in a user interface and need not be the same as `OperationName`.
```

Comment:
can we add:

If the DisplayName property not set before, it will return the same value as OperationName